packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -55,7 +54,66 @@ describe('app-generator (SLOW)', function () {
         .runShell('npm', ['test'], {
           // Disable stdout
           stdio: [process.stdin, 'ignore', process.stderr],
-          cwd: sandbox,
+          cwd: appProps.outdir,
+        })
+        .on('close', code => {
+          assert.equal(code, 0);
+          resolve();
+        });
+    });
+  });
+
+  after(function () {
+    // Increase the timeout to accommodate slow CI build machines
+    // eslint-disable-next-line no-invalid-this
+    this.timeout(30 * 1000);
+
+    process.chdir(rootDir);
+    build.clean(['node', 'run-clean', appProps.outdir]);
+    process.chdir(process.cwd());
+  });
+});
+
+describe('app-generator with Yarn (SLOW)', function () {
```

Comment:
Is it going to fail if no `yarn` is installed on the test machine?

Reply:
Yes, Lerna will not detect yarn.
![Screenshot 2020-05-03 at 19 36 15](https://user-images.githubusercontent.com/13822438/80921233-6990aa80-8d75-11ea-9c90-3d59cdd77812.png)

Reply:
Let's make the test conditional on the presence of `yarn` then.

Reply:
Done